view/frontend/layout/algolia_search_handle.xml

```diff
@@ -10,6 +10,7 @@
         <css src="Algolia_AlgoliaSearch::internals/autocomplete.css" />
         <css src="Algolia_AlgoliaSearch::internals/grid.css" />
         <css src="Algolia_AlgoliaSearch::internals/algolia-reset.css" />
+        <css src="Algolia_AlgoliaSearch::internals/algoliasearch.css" />
```

Comment:
Would it be possible to port your CSS changes into the autocomplete stylesheet? We removed the algoliasearch.css file in our v2.x update.

Reply:
Thanks for the review. I can see now that this line was removed in #838, but the corresponding file was never removed. I'll move the relevant styles to `view/frontend/web/internals/autocomplete.css` as suggested and remove this file to avoid future confusion.